hack/verify_pipeline.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+set -ue
+
```

Comment:
Maybe some commentary here to explain what this script is for and how to use it? Perhaps it include typical commands, e.g.

```
    # Use the policies defined in the Enterprise Contract to validate a pipeline definition.
    #
    # Typical usage:
    #   oc get pipeline somepipeline -n somenamespace -o json > pipeline.json
    #   ./verify_pipeline.sh pipeline.json
    #